cdragontoolbox/export.py

```diff
@@ -536,16 +537,49 @@ def convert_to_file(self, fin, fout):
             # "OSError: cannot identify image file" happen for some files with a wrong extension
             raise FileConversionError("cannot convert image to PNG")
 
+
 class BinConverter(FileConverter):
-    def __init__(self, regex):
+    def __init__(self, regex=None, extensions=None):
```

Comment:
The `extensions` parameter is not needed; it overlaps with `regex` and its use can be replaced by using an appropriated `regex` value.
You only use it in tests, in which you could use `BinConverter(re.compile(r"\.bin$"))`

Reply:
Removed from the updated version.